src/stories/components/button/big-button/big-button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Button } from '@mui/material';
+
+interface Props {
+  onClick?: () => void;
+  title?: string
+  color?: string
+}
+
+const BigButton = ({ onClick, title = 'Back' }: Props) => {
+  return (
+    <StyledBigButton onClick={onClick} >{title}</StyledBigButton>
+  );
+};
+
+const StyledBigButton = styled(Button)`
+min-width: 512px;
+height: 32px;
+border:1px solid rgba(37, 39, 61, 0.75);
+border-radius: 8px;
+font-family: 'Inter, sans-serif';
```

Comment:
This shouldn't proc the right font because of incorrect use of the quotes. You can delete it.

Reply:
done